<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Restaurant &quot;La Tia Veneno&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Platillo :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Precio   :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cantidad :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPlatillo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrecio">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbCantidad">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POSTRES :</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPostre1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Arroz con leche</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPostre3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mazamorra Morada</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPostre2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Leche Azada</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPostre4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Crema Volteada</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="txtSalida">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcesar">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Procesar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comprobante : </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbBoleta">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Boleta</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFactura">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Factura</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
